Source/Krypton Components/Krypton.Workspace/Controls Workspace/KryptonWorkspace.cs

```diff
@@ -2804,7 +2804,7 @@ internal Rectangle SeparatorMoveBox(ViewDrawWorkspaceSeparator separator)
                                      Point splitter)
         {
             // Get the sequence that contains the items moved
-            var parentSequence = (KryptonWorkspaceSequence)separator.WorkspaceItem.WorkspaceParent;
+            var parentSequence = separator.WorkspaceItem.WorkspaceParent as KryptonWorkspaceSequence;
```

Comment:
If the var is just going to be ignoring the nullable possibility from using the `as`
then use a cast, and remove the `!` usage later on.